strutil/map_test.go

```diff
@@ -81,7 +81,7 @@ k: v
 `)
 	var om strutil.OrderedMap
 	err := yaml.Unmarshal(yamlStr, &om)
-	c.Assert(err, ErrorMatches, `found duplicate key "k"`)
+	c.Assert(err, ErrorMatches, `(?ms).*mapping key "k" already defined at line 2`)
```

Comment:
I would leave the line number out of the check, just in case they decide to change the message to "...already defined at line 0", but that's just me -- no need to change :-)